addon/components/au-link-external.js

```diff
@@ -3,24 +3,42 @@ import Component from '@glimmer/component';
 const SKIN_CLASSES = {
   primary: 'au-c-link',
   secondary: 'au-c-link au-c-link--secondary',
+  button: 'au-c-button au-c-button--primary',
+  'button-secondary': 'au-c-button au-c-button--secondary',
+  'button-naked': 'au-c-button au-c-button--naked',
 };
 
 export default class AuLinkExternal extends Component {
-  get skin() {
+  get skinClass() {
     if (SKIN_CLASSES[this.args.skin]) {
       return SKIN_CLASSES[this.args.skin];
     } else {
       return SKIN_CLASSES.primary;
     }
   }
 
-  get width() {
-    if (this.args.width == 'block') return 'au-c-link--block';
+  get widthClass() {
+    if (this.args.width == 'block')
+      if (this.args.skin.startsWith('button')) return 'au-c-button--block';
+      else return 'au-c-link--block';
+    else return '';
+  }
+
+  get activeClass() {
```

Comment:
I don't think we need an "active" state for an external link? At least not one based on an argument.

Reply:
In the previous state of the component this did nothing because no string was bound to `this.active` so I assumed that this was a bug? Plus the fact that the active state was also toggle-able (but with no visual effect) inside Storybook. Maybe @Dietr can enlighten us?

Reply:
I assumed it was a copy pasta mistake from the AuLink component actually 😄.

Reply:
Well, that also makes sense 😅 I'll remove this.